### ISSUE TITLE
Invalid value in foreach in debug_data()

### ISSUE THREAD

**Issue by paulschreiber** [WordPress/health-check#388]
### Feature request/bug description

I'm seeing this warning:
```
Warning: Invalid argument supplied for foreach() in /home/customer/www/empirejustice.org/public_html/
wp-content/plugins/health-check/includes/class-health-check-debug-data.php on line 59
```

related code:
```
  59                 foreach ( $core_updates as $core => $update ) {
  60                         if ( 'upgrade' === $update->response ) {
  61                                 // translators: %s: Latest WordPress version number.
  62                                 $core_update_needed = ' ' . sprintf( __( '(Latest version: %s)', 'health-chec     k' ), $update->version );
  63                         } else {
  64                                 $core_update_needed = '';
  65                         }
  66                 }
```

Debugging, I see `$core_updates` is `false`. I suspect this is because of Siteground's own updater.

**Comment by paulschreiber** [comment]
Adding an `if ( is_array( $core_updates ) ) {` check avoids the problem.

**Comment by paulschreiber** [comment]
Core ticket: https://core.trac.wordpress.org/ticket/51818

**Comment by Clorith** [comment]
Fixed in d5ba5b6adf3641566f6a4f4695dec9d222ab5ba6
